x11-terms/tilda/tilda-1.5.2.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
```

Comment:
I wouldn't bother changing the stable ebuild for no reason given we're bumping anyway.